src/scripts/main.js

```diff
@@ -1 +1,57 @@
 'use strict';
+
+const root = document.body;
+
+// firstPromise
+new Promise((resolve, reject) => {
```

Comment:
```suggestion
const firstPromise = new Promise((resolve, reject) => {
```
Just create variables for promises and won't need the comments